ansible_events_ui/key.py

```diff
@@ -0,0 +1,34 @@
+import asyncio
+import logging
+import os
+import shutil
+import tempfile
+
+ssh_keygen = shutil.which("ssh-keygen")
+logger = logging.getLogger()
+
+
+async def generate_ssh_keys():
+
+    local_working_directory = tempfile.mkdtemp()
+    try:
```

Comment:
Minor: The context manager usage here is more idiomatic:
```suggestion
    with tempfile.TemporaryDirectory as local_working_directory:
```